.circleci/config.yml

```diff
@@ -206,6 +206,8 @@ commands:
         type: string
     steps:
       - checkout
+      - build/determinator:
+          paths: "orc8r lte cwf feg"
```

Comment:
this should be a param, the lte integ tests only need to run for for `orc8r lte` for e.g.